pegasus/router.rb

```diff
@@ -288,6 +288,9 @@ def self.load_configs_in(dir)
       response.headers['X-Frame-Options'] = 'ALLOWALL'
     end
 
+    if @locals[:header]['content-type']
```

Comment:
Apologies for my Pegasus ignorance, but does @locals[:header] refer only to the block at the beginning of a haml file such as the one you've modified below in records.haml?